src/main/java/org/aszjch/demoapp/aplication/ArticleDto.java

```diff
@@ -1,11 +1,17 @@
 package org.aszjch.demoapp.aplication;
 
 import jakarta.validation.constraints.Null;
+import lombok.Getter;
+import lombok.Setter;
 
+@Getter
+@Setter
 class ArticleDto {
 
-  @Null(message = "id must not be provided.")
-  private Long id;
-
+    @Null(message = "id must not be provided.")
```

Comment:
Zamiast walidacji na null, może lepiej dać @JsonIgnore? wtedy na pewno to nie wpadnie z JSON'a, a skoro to zawsze ma być null to może nie ma potrzeby tego tak zostawiać?